src/main/java/seedu/address/model/util/Time.java

```diff
@@ -0,0 +1,87 @@
+package seedu.address.model.util;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+public class Time implements Comparable<Time> {
+    public static final String TIME_REGEX = "(([0-1]\\d)|(2[0-3])):([0-5]\\d)";
+    public static final String MESSAGE_CONSTRAINTS =
```

Comment:
Same as above I think its better to shift this message into to class and say To should be in the format ...